src/Executor/Executor.php

```diff
@@ -18,6 +18,8 @@ class Executor
     const METHOD_CONNECT = 'CONNECT';
     const METHOD_TRACE = 'TRACE';
 
+    const DEFAULT_HOST = 'http://appwrite-executor/v1';
```

Comment:
No need for the default value here, and if missing we can just throw an exception instead of fixing a maintainer mistake.